src/TinyPubSub/TinyPubSub.cs

```diff
@@ -334,6 +346,7 @@ public static TinyEventArgs PublishControlled<T>(string channel, T instance, Act
                     if (subscription.Owner.Target == null)
                     {
                         subscriptionsToRemove.Add(subscription);
+                        continue;
```

Comment:
Why continue here? What happens if ActionWithArgumentAndArgs isn't null? Then it will not be called?

Reply:
Because if it is null the object owning it is been collected by GC. And then we can remove that subscription